### PR TITLE
AquaComms

### PR DESCRIPTION
**Pull request by carbonarok**
New provider and parser added for AquaComms.

### CODE REVIEW

File: tests/unit/data/aquacomms/aquacomms1_result.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        "account": "999",
+        "circuits": [
+            {
+                "circuit_id": "111-AAA-11-11BB-33333",
+                "impact": "OUTAGE"
+            }
+        ],
+        "end": 1602572400,
+        "maintenance_id": "11111",
+        "organizer": "support@cogentco.com",
```
Comment (chadell) [inline tests/unit/data/aquacomms/aquacomms1_result.json:12]: cogent?

File: circuit_maintenance_parser/provider.py [hunk 2]
```diff
@@ -105,6 +106,15 @@ def get_provider_type(cls) -> str:
 ####################
 
 
+class AquaComms(GenericProvider):
+    """AquaComms provider custom class."""
+
+    _processors: List[GenericProcessor] = [
+        CombinedProcessor(data_parsers=[EmailDateParser, HtmlParserAquaComms1, SubjectParserAquaComms1]),
+    ]
+    _default_organizer = "support@cogentco.com"
```
Comment (chadell) [inline circuit_maintenance_parser/provider.py:115]: wrong?

File: circuit_maintenance_parser/parsers/aquacomms.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+class SubjectParserAquaComms1(EmailSubjectParser):
+    """Parser for Seaborn subject string, email type 1."""
+
+    def parse_subject(self, subject):
+        """Parse subject of email file.
+
+        Subject: Aqua Comms Planned Outage Work ISSUE=111111 PROJ=999
+        """
+        data = {}
+        search = re.search(r".+ISSUE=([0-9]+).PROJ=([0-9]+)", subject)
```
Comment (glennmatthews) [inline circuit_maintenance_parser/parsers/aquacomms.py:22]:
```suggestion
        search = re.search(r"ISSUE=([0-9]+).+PROJ=([0-9]+)", subject)
```

File: tests/unit/data/aquacomms/aquacomms1.eml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+vetica, sans-serif" size=3D"2">Please note that all time stamps below will =
+appear in US Central time.<br><br></font></td>
+      </tr>
+     =20
+       =20
+        <tr>
+          <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, H=
+elvetica, sans-serif" size=3D"2">- Update - There are no new updates.<br></=
+font></td>
+        </tr>
```
Comment (glennmatthews) [inline tests/unit/data/aquacomms/aquacomms1.eml:73]: Should this go into the maintenance `summary` field instead of leaving it blank?

**Comment by glennmatthews** [comment]
Talk to me about that last commit, please - why *shouldn't* we be reading the timezone when parsing the email?

File: circuit_maintenance_parser/parsers/aquacomms.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
+"""Turkcell parser."""
```
Comment (chadell) [inline circuit_maintenance_parser/parsers/aquacomms.py:1]:
```suggestion
"""AquaComms parser."""
```

**Review by chadell — APPROVED** [review]
LGTM

**Review by glennmatthews — DISMISSED** [review]

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**